kiwixbuild/dependencies/all_dependencies.py

```diff
@@ -13,10 +13,10 @@ class Builder(NoopBuilder):
         @classmethod
         def get_dependencies(cls, platformInfo, allDeps):
             base_deps = ['zlib', 'lzma', 'zstd', 'xapian-core', 'pugixml', 'libcurl', 'icu4c', 'mustache', 'libmicrohttpd']
-            # zimwriterfs
+            # zimwriterfs from zim-tools
             if platformInfo.build not in ('android', 'win32'):
                 base_deps += ['libmagic', 'gumbo']
-            
+
             # zimtools
             if platformInfo.build not in ('android','iOS'):
                 base_deps += ['docoptcpp']
```

Comment:
We should "merge" this tests.
zimwriterfs is part of zimtools. We should only have a specific case for win32 as we don't compile it on windows.